### DISCUSSION TITLE
P9752 [CSP-S 2023] 密码锁 题解 — scp020 的博客

### DISCUSSION THREAD

**Discussion by giscus** [scp020/scp020.github.io#4] [General]
# P9752 [CSP-S 2023] 密码锁 题解 — scp020 的博客

为什么考场上一大堆人读错题了啊。顺便祝提高组出题人身体健康。

https://scp020.github.io/2023/10/22/solution-p9752/

**Comment by StevehimYu** [comment]
tql
